nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/java/org/nuxeo/ecm/platform/comment/impl/TreeCommentManager.java

```diff
@@ -166,10 +177,16 @@ public Comment createComment(CoreSession s, Comment comment) {
                 Comments.externalEntityToDocumentModel((ExternalEntity) comment, commentDocModel);
             }
 
+            // Create the comment document model
             commentDocModel = session.createDocument(commentDocModel);
+            Comment createdComment = Comments.newComment(commentDocModel);
+
+            // Manage the related text of the top level document
```

Comment:
the method names speak by themself the comment are useless (the previous one also)

Reply:
I told myself the same thing ;)

Reply:
finally i did it :)